### PR TITLE
Adds unit test for unit conversions

### PR DESCRIPTION
**Pull request by CoryMartin-NOAA**
Will be the first test the repo runs in CI; also works as an example on how to convert units using emcpy

### CODE REVIEW

**Comment by aerorahul** [comment]
ok. we need to add an action to trigger the test.

File: tests/test_convert_units.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import numpy as np
+import emc.calcs.units
```
Comment (aerorahul) [inline tests/test_convert_units.py]: `emc` everywhere should become `emcpy`

**Review by aerorahul — CHANGES_REQUESTED** [review]
change `emc` to `emcpy`

File: tests/test_convert_units.py (hunk 1 above)
Comment (CoryMartin-NOAA) [reply tests/test_convert_units.py]: Changed, please re-review

File: tests/test_convert_units.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import emcpy.calcs.units
```
Comment (aerorahul) [inline tests/test_convert_units.py]: may be call the script `test_units.py`.

**Review by aerorahul — APPROVED** [review]

**Pull request merged**